.teamcity/_self/projects/WebApp.kt

```diff
@@ -329,7 +329,6 @@ object RunAllUnitTests : BuildType({
 	artifactRules = """
 		test_results => test_results
 		artifacts => artifacts
-		checkstyle_results => checkstyle_results
```

Comment:
**question:non-blocking** - is this removed because any failures are now reported properly in the Tests tab for a run?

Reply:
Actually just because eslint wasn't being run in this test suite! It has its own build right now, so this file hasn't been used in a while